data/smc/missing researchers.txt

```diff
@@ -0,0 +1,2174 @@
+	names "free worlds capital"
+	cargo 1
+	personality
+		heroic disables frugal opportunistic
+	variant 3
+		"Dreadnought"
+	variant 3
+		"Skein"
+		"Finch" 6
+	variant 3
+		"Skein"
+		"Finch" 6
+	variant 3
+		"Roost"
+		"Finch" 4
+	variant 10
+		"Bastion"
+	variant 5
+		"Bastion (Heavy)"
+	variant 5
+		"Bastion (Laser)"
+	variant 8
+		"Osprey"
+	variant 4
+		"Osprey (Missile)"
+	variant 4
+		"Osprey (Laser)"
+	variant 6
+		"Bastion"
+		"Osprey"
+	variant 3
+		"Bastion (Heavy)"
+		"Osprey (Missile)"
+	variant 3
+		"Bastion (Laser)"
+		"Osprey (Laser)"
+	variant 2
+		"Falcon"
+	variant 1
+		"Falcon (Heavy)"
+	variant 1
+		"Falcon (Laser)"
+	variant 4
+		"Falcon"
+		"Bastion"
+	variant 2
+		"Falcon (Heavy)"
+		"Bastion (Heavy)"
+
+fleet "Hai Wormhole Guards"
+	government "Hai"
+	names hai
+	cargo 0
+	personality heroic
+	variant 7
+		"Shield Beetle (Police)"
+	variant 7
+		"Shield Beetle (Tracker)"
+	variant 7
+		"Shield Beetle (Pulse)"
+	variant 7
+		"Shield Beetle"
+	variant 4
+		"Lightning Bug" 3
+	variant 4
+		"Lightning Bug (Pulse)" 3
+	variant
+		"Grasshopper" 5
+	variant
+		"Grasshopper (Tracker)" 5
+
+event "hai and human ships at wormhole"
+	system "Waypoint"
+		add fleet "Hai Wormhole Guards" 400
+	system "Heia Due"
+		add fleet "Hai Wormhole Guards" 1000
+	system "Ultima Thule"
+		add fleet "Navy Wormhole Guards" 600
+		add fleet "Syndicate Wormhole Guards" 1000
+		add fleet "Free Worlds Wormhole Guards" 1500
+		add fleet "Deep Security Wormhole Guards" 1500
+		# Occasional pirates to show the fleet in action
+		add fleet "Small Northern Pirates" 5000
+		add fleet "Small Core Pirates" 5000
+		# Smugglers would be bright enough not to try this route
+		remove fleet "Smuggler (Hai Traffickers)"
+	system "Rajak"
+		add fleet "Navy Wormhole Guards" 1500
+		add fleet "Syndicate Wormhole Guards" 2500
+		add fleet "Free Worlds Wormhole Guards" 3500
+		add fleet "Deep Security Wormhole Guards" 3500
+		# Smugglers would be bright enough not to try this route
+		remove fleet "Smuggler (Hai Traffickers)"
+
+mission "Hai Rescue [B02] Diplomacy: Syndicate"
+	landing
+	name "Missing Researchers"
+	description "Visit <waypoints> then escort Teeneep's Centipede to <destination>. Payment is <payment>."
+	source "Hai-home"
+	waypoint "Waypoint"
+	waypoint "Ultima Thule"
+	destination "Sunracer"
+	on visit
+		dialog `To finish this mission, you must visit Waypoint and Ultima Thule, and then land with Teeneep's Centipede.`
+	to offer
+		has "Hai Rescue [B01] Election: done"
+	on enter "Waypoint"
+		dialog `There is a massive fleet of Hai ships here, and more coming in all the time. If this fleet were to enter human space, we might be overrun.`
+	on enter "Ultima Thule"
+		dialog `Although it is reassuring to see humans come together for a common cause, you don't look forward to seeing two species enter an all-out war over a misunderstanding. You also wonder how the human authorities are keeping the Hai a secret with 
```

Comment:
I fixed most of those, but unless I'm gravely mistaken, you don't capitalize in this situation:

- "My feet are green," he said, "because I never wash them."

The "b" in "because" is not capitalized since the sentence has not yet ended.